projects/knora-ui/src/lib/viewer/values/time-value/time-value.component.html

```diff
@@ -0,0 +1,22 @@
+<span [formGroup]="form" class="time-host-component">
+    <mat-form-field class="large-field child-value-component">
+      <kui-time-input #timeInput [formControlName]="'timeValue'" class="value" placeholder="Time value" [readonly]="mode === 'read'" [errorStateMatcher]="matcher"></kui-time-input>
+      <mat-error *ngIf="form.controls.timeValue.hasError('valueNotChanged')">
+        Please change the value
+      </mat-error>
+    </mat-form-field>
```

Comment:
Could you add some space or a line break  between the value and comment? 

in Chrome:
<img width="466" alt="Screenshot 2020-03-12 at 12 03 36" src="https://user-images.githubusercontent.com/6000023/76515229-8be30780-6459-11ea-8663-6358106a3dab.png">

Reply:
<img width="811" alt="Screen Shot 2020-03-12 at 13 46 49" src="https://user-images.githubusercontent.com/60604010/76523021-f4d17c00-6467-11ea-99fa-1547b49dad66.png">


It looks okay on my screen but I know the css needs some improvement. I'll move the css to a new PR and add a line break as the last thing before merging.